piano/app/src/main/res/layout/fragment_white_key.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".WhiteKeyFragment">
+
+
+    <Button
+        android:id="@+id/whiteKey"
+        android:layout_width="59dp"
+        android:layout_height="233dp"
```

Comment:
Samme som på svarte keys, gjøre høyden mer dynamisk for å kunne fungere på ulike skjermstørrelser